conversions/pair_to_number_conversion.py

```diff
@@ -0,0 +1,25 @@
+class pair_to_number_conversion():
+    """
+    Purpose: This class contains the method to convert pair
+            number to the color for a 25-pair color code.
+            Example: MAJOR_COLOR = WHITE & MINOR_COLOR = BROWN
+            PAIR NUMBER = 4
+    """
+    def __init__(self, MAJOR_COLORS, MINOR_COLORS) -> None:
+        self.MAJOR_COLORS = MAJOR_COLORS
+        self.MINOR_COLORS = MINOR_COLORS
+
+    def test_pair_to_number(self, major_color, minor_color, expected_pair_number):
```

Comment:
Test code should be kept separately from production code. the test code should not be part of the shipped software.

Reply:
Sorry for the late response. Yes you can make changes and commit :-)

Reply:
Thank you very much, I have uploaded again for review. Can you please let me know if it looks good in the latest commit?

Reply:
👍